.env

```diff
@@ -36,6 +36,7 @@ RABBITMQ_URL=amqp://guest:guest@localhost:5672
 # For a generic SMTP server, use: "smtp://localhost:25?encryption=&auth_mode="
 # Delivery is disabled by default via "null://localhost"
 MAILER_URL=null://localhost
+BCC_MAILER_ADDR='email@domain.null'
```

Comment:
Should this be moved to line 56, to be with the other variables regarding the mailer? Maybe even put them in a ### block. Also named the same? If so, maybe something like "MAILER_BCC_ADDR" or "MAILER_BCC_TO" ?

Reply:
Done